src/components/mainPages/SidebarPage.vue

```diff
@@ -181,19 +260,21 @@ export default {
         title="sign in"
       />
     </div>
-    <div class="sidebar-buttons">
+    <div
+      class="sidebar-buttons"
+    >
       <div class="mode-buttons">
         <ButtonText
           v-for="mode in sidebarModes"
           :key="mode"
           :class="{
-            'mode-button' : !isMobileScreen && mode !== 'bin',
+            'mode-button' : !isMobileScreen && !compactLayout && mode !== 'bin',
```

Comment:
better to put this condition into a computed
also because you use it twice
but even it is once – it's gonna be more readable and template more clean